libs/sdk-ui-ext/src/internal/dashboardEmbedding/utils/format.ts

```diff
@@ -0,0 +1,8 @@
+// (C) 2007-2021 GoodData Corporation
+
+// Metric format is considered to represent value in percent when
+// A) format string has no conditional separators (i.e. no semicolons
+//    except a single one at the end); otherwise too difficult to parse :-(
+// B) percentage symbol is found (not directly preceded by backslash)
+export const isMetricFormatInPercent = (format = ""): boolean =>
+    /^([^;]*)%([^;]*)[;]*$/.test(format.trim().replace(/\\%/g, ""));
```

Comment:
small stuff: this looks useful outside of dashboard embedding. perhaps move to a more common / lower level module. it is likely that we already have some code elsewhere that does testing of percent formatting

Reply:
indeed. chartAxes.ts / isPercentage

Reply:
Moved to sdk-model, thanks for the pointer!

Reply:
the chart axes version is too simplistic imo, this thing takes also conditional formatting into account

Reply:
I could try to replace that with the new function, but I'm a bit scared tbh

Reply:
Replaced this in a separate commit, hopefully it does not break something 🤞